app/assets/stylesheets/incurred_incidentals/form.css

```diff
@@ -0,0 +1,13 @@
+.multi-upload-desc {
+  width: inherit !important;
```

Comment:
Are we sure that these `!important`s are necessary?

I ask because it's pretty easy to get into an arms race with those.